src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,6 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(Lottery.getRandomBall());
```

Comment:
To test the program, in` main()` method, create **three** balls using `getRandomBall() `method from `Lottery` class. And print information about them in the console.